lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -260,17 +263,19 @@ def cmd_banner(*args)
   def cmd_tip_help
     print_line "Usage: tip [options]"
     print_line
-    print_line "Print a useful tip on how to use Metasploit"
+    print_line "Print a useful list of productivity tips on how to use Metasploit"
     print @@tip_opts.usage
   end
 
+  alias cmd_tips_help cmd_tip_help
```

Comment:
This reinforces my point I made above; if we are using aliases and depreciating `tip`, then we should update the usage guide to use `tips` rather than `tip`.

Reply:
Fixed with latest commit.